config.yml

```diff
@@ -7723,3 +7441,355 @@ production_recipes:
+    inputs:
+      Nether Wart:
+        material: NETHER_STALK
+        amount: 1280
+      Glass Bottle:
+        material: GLASS_BOTTLE
+        amount: 6
+    outputs:
+      Exp Bottle:
+        materials: EXP_BOTTLE
+        amount: 6
+  Iron_Crate:
+    name: Package Iron Crate
+    production_time: 40
+    inputs:
+      Wood Crate:
+        material: STORAGE_MINECART
+        amount: 4
+        display_name: Wood Crate
+        lore: Packaged local market goods. Useless for resource extraction without further refinement.
+      Vine:
+        material: VINE
+        amount: 640
+      Clay Blocks:
+        material: CLAY
+        amount: 64
+      Cake:
+        material: CAKE
+        amount: 16
+      Red Mushroom:
+        material: RED_MUSHROOM
+        amount: 64
+      Apple:
+        material: APPLE
+        amount: 64
+      Netherrack:
+        material: NETHERRACK
+        amount: 96
+      Flint:
+        material: FLINT
+        amount: 64
+    outputs:
+      Iron Crate:
+        material: POWERED_MINECART
+        amount: 1
+        display_name: Iron Crate
+        lore: Processed market supplies. It can be converted into XP but it's not very efficient...
+  Iron_XP_Recipe_1:
+    name: Craft XP Bottles 1
+    production_time: 10
+    inputs:
+      Bread:
+        material: BREAD
+        amount: 640
+      Netherwart:
+        material: NETHER_STALK 
+        amount: 320
+      Baked Potato:
+        material: BAKED_POTATO
+        amount: 640
+      Glass Bottle:
+        material: GLASS_BOTTLE
+        durability: 16
+        amount: 18
+    outputs:
+      Exp Bottle:
+        materials: EXP_BOTTLE
+        amount: 18
+  Iron_XP_Recipe_2:
+    name: Craft XP Bottles 2
+    production_time: 10
+    inputs:
+      Rations:
+        material: MUSHROOM_SOUP
+        amount: 16
+        display_name: Rations
+        lore: An item used to create a Bastion Block
+      Carrots:
+        material: CARROT
+        amount: 512
+      Sugar:
+        material: SUGAR
+        amount: 640
+      Glass Bottle:
+        material: GLASS_BOTTLE
+        amount: 13
+    outputs:
+      Exp Bottle:
+        materials: EXP_BOTTLE
+        amount: 13
+  Iron_XP_Recipe_3:
+    name: Craft XP Bottles 3
+    production_time: 10
+    inputs:
+      Cactus:
+        material: CACTUS
+        amount: 256
+      Cocoa:
+        material: INK_SACK
+        durability: 3
+        amount: 256
+      Baked Potato:
+        material: BAKED_POTATO
+        amount: 320
+      Glass Bottle:
+        material: GLASS_BOTTLE
+        amount: 16
+    outputs:
+      Exp Bottle:
+        materials: EXP_BOTTLE
+        amount: 16
+  Diamond_Crate:
+    name: Package Diamond Crate
+    production_time: 20
+    inputs:
+      Iron Crate:
+        material: POWERED_MINECART
+        amount: 2
+        display_name: Iron Crate
+        lore: Processed market supplies. It can be converted into XP but it's not very efficient...
+      Golden Carrot:

```

Comment:
I think it'd be interesting to see a more serious use out of gold than just the standard tools or pyramid bases that people only use them for now. It is grindy, yes, but people use the portal farms already for other competitive goods, so I think that by adding in a 3rd opportunity for the nuggets to be utilized, we'll see players potentially even starting to use the gold ore smelting more than previous.